lib/Scheduler/sched_region.cpp

```diff
@@ -244,6 +257,10 @@ FUNC_RESULT SchedRegion::FindOptimalSchedule(
 #endif
   }
 
+  // Log the lower bound on the cost, allowing tools reading the log to compare
+  // absolute rather than relative costs.
+  Logger::Info("Lower bound of cost before scheduling: %d", costLwrBound_);
```

Comment:
Is there a reason why this was moved away from the lower bound cost calculation as opposed to being directly after it?

Reply:
No, I must have accidentally placed it too far when I was making this change. There's should be no reason as to why this is so far below the lower bound calculation itself.

Reply:
Okay, good to know.